docs/pages/material-ui/api/table-pagination.json

```
@@ -8,6 +8,20 @@
     "backIconButtonProps": { "type": { "name": "object" } },
     "classes": { "type": { "name": "object" } },
     "component": { "type": { "name": "elementType" } },
+    "components": {
+      "type": {
+        "name": "shape",
+        "description": "{ first?: elementType, last?: elementType, next?: elementType, previous?: elementType }"
```

```suggestion
        "description": "{ First?: elementType, Last?: elementType, Next?: elementType, Previous?: elementType }"
```

This is the convention we use at this moment.